tests/Makefile

```diff
@@ -1,66 +1,14 @@
 SRC_DIR = ../
 
-TESTS = \
-	sdl/test_model.py \
-	promela/test_model.py \
-	translator/test_translator.py \
-	stop_condition/test_scl_parser.py
-
 .PHONY : \
 	check \
 	unit \
-	integration \
-	parameterless \
-	simpleparameters \
-	flowcontrol \
-	stop_condition_eventually \
-	stop_condition_never_filter_out \
-	stop_condition_present \
-	stop_condition_length \
-	stop_condition_exist \
-	integration_tests
+	integration
 
-unit: ${TESTS}
-	PYTHONPATH=${SRC_DIR} python3 -m pytest ${TESTS} -vv
+unit:
+	${MAKE} -C $@ check
```

Comment:
How does it work?

Reply:
it executes the commans: `make -C unit check`, which means: run make in directory 'unit' using target check. If the make fails then the parent make stops (with additional error message).